engines/dfc_provider/app/serializers/dfc_provider/product_serializer.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Serializer used to render the products passed
+# into JSON-LD format based on DFC ontology
+module DfcProvider
+  class ProductSerializer
+    def initialize(products, base_url)
+      @products = products
+      @base_url = base_url
+    end
+
+    def serialized_json
+      {
+        "@context" =>
+        {
+          "DFC" => "http://datafoodconsortium.org/ontologies/DFC_FullModel.owl#",
+          "@base" => @base_url
+        },
+        "@id" => "/enterprise/products",
+        "DFC:supplies" => serialized_products
+      }.to_json
```

Comment:
In this `#serialized_json` method, `#to_json` is called explicitly, and then we're calling `render json: products_json` as well in the controller. I don't think we need to do both, `render json:` will already do that, right? So I think we can remove `#to_json` here.

Reply:
Well, the `serialized_json` method is supposed to return JSON so I think it is good to keep it this way, no? Or should I rename the method? `serialized_hash` ?

Reply:
Ok, I have put `serialized_hash` as it is not JSON now.